nixos/modules/services/networking/bind.nix

```diff
@@ -211,25 +212,33 @@ in
 
     networking.resolvconf.useLocalResolver = mkDefault true;
 
-    users.users.${bindUser} =
-      {
+    users = {
+      groups = {
+        "${bindGroup}" = {
+          gid = config.ids.gids.bind;
+        };
+      }
+      users.${bindUser} = {
         uid = config.ids.uids.bind;
+        groups = "${bindGroup}";
         description = "BIND daemon user";
       };
+    };
 
     systemd.services.bind = {
       description = "BIND Domain Name Server";
       after = [ "network.target" ];
       wantedBy = [ "multi-user.target" ];
 
       preStart = ''
-        mkdir -m 0755 -p /etc/bind
+        mkdir -m 7600 -p /etc/bind
+        chown root:${bindGroup} /etc/bind
         if ! [ -f "/etc/bind/rndc.key" ]; then
           ${pkgs.bind.out}/sbin/rndc-confgen -c /etc/bind/rndc.key -u ${bindUser} -a -A hmac-sha256 2>/dev/null
         fi
 
         ${pkgs.coreutils}/bin/mkdir -p /run/named
-        chown ${bindUser} /run/named
+        chown root:${bindGroup} /run/named
```

Comment:
This should be RuntimeDirectory in the serviceConfig.

Reply:
Seems irrelevant to this MR? I am only integrating rights for user group here?

Reply:
It's tangent to this MR, hence the "should".

Reply:
> This should be RuntimeDirectory in the serviceConfig.

`RuntimeDirectory` will create the directory as the user the service runs as, which in this case is `root` :disappointed:

Reply:
Well, creating the RuntimeDirectory should be in serviceConfig. Modifying it's chmod/chown can be done in preStart. But I'm not yet convinced the use case is sensible.

Reply:
> I'm not yet convinced the use case is sensible.

Agreed.

Reply:
submitted commit with integrated RuntimeDirectory